pkg/scheduler/framework/plugins/interpodaffinity/scoring_test.go

```diff
@@ -561,7 +561,7 @@ func TestPreferredAffinity(t *testing.T) {
 		{
 			name:       "invalid Affinity fails PreScore",
 			pod:        &v1.Pod{Spec: v1.PodSpec{NodeName: "", Affinity: invalidAffinityLabels}},
-			wantStatus: framework.NewStatus(framework.Error, "invalid label value"),
+			wantStatus: framework.NewStatus(framework.Error, `Invalid value: "{{.bad-value.}}"`),
```

Comment:
Follow up to change to cmp.Diff here too.

Reply:
`PreScore` formats the error to a string. It's a little hard to use `cmp.Diff` to validate the error type `invalid value` because the returned status contains only strings instead of structural value. To do that (ensure error type is `invalid value`), the current implementation is `strings.Contains`.